README.md

```diff
@@ -15,6 +15,12 @@
         <img src="https://circleci.com/gh/vapor/jwt.svg?style=shield" alt="Continuous Integration">
     </a>
     <a href="https://swift.org">
-        <img src="http://img.shields.io/badge/swift-3.1-brightgreen.svg" alt="Swift 3.1">
+        <img src="http://img.shields.io/badge/swift-4.1-brightgreen.svg" alt="Swift 4.1">
     </a>
 </p>
+
+<hr>
+
+**Original author**
+
+- Siemen Sikkema, [@siemensikkema](http://github.com/siemensikkema)  
```

Comment:
@siemensikkema I moved the attribution down here to better match our other repos. Let me know if you approve 👍

Reply:
@tanner0101 Go ahead :)